.github/renovate.json5

```
@@ -389,6 +389,15 @@
         "v1.13"
       ]
     },
+    {
+      // Avoid updating patch version for statedb as it might contain breaking changes
+      "matchPackageNames": [
+        "github.com/cilium/statedb",
+      ],
+      "matchUpdateTypes": [
+        "patch",
+      ],
+    },
```

We should do the opposite otherwise we will end-up with a PR that only updates statedb. By doing like this we will still make sure that statedb patch versions will be grouped with other dependencies.
```suggestion
      // Avoid updating patch version for statedb as it might contain breaking changes
      "enabled": false,
      "matchPackageNames": [
        "github.com/cilium/statedb",
      ],
      "matchUpdateTypes": [
        "major",
        "minor",
        "pin",
        "pinDigest",
        "digest",
        "lockFileMaintenance",
        "rollback",
        "bump",
        "replacement",
      ],
    },
```